packages/compass-e2e-tests/helpers/commands/set-feature.ts

```diff
@@ -1,15 +1,14 @@
 import type { CompassBrowser } from '../compass-browser';
+import { preferences } from 'compass-preferences-model';
 
 export async function setFeature(
   browser: CompassBrowser,
   name: string,
   value: boolean | string
 ): Promise<void> {
   await browser.execute(
-    (_name, _value) => {
-      const preferences = (global as any)?.hadronApp?.preferences;
-      preferences.set(_name, _value);
-      preferences.save();
+    async (_name, _value) => {
+      await preferences.savePreferences({ [_name]: _value });
```

Comment:
This is running in a different context, you won't be able to pass the preferences model here like that

Reply:
A very good point 😏